src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/WindowsSecurityToken.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using System.Security.Principal;
+using System.ServiceModel;
+
+namespace System.IdentityModel.Tokens
+{
+
+    public class WindowsSecurityToken : SecurityToken, IDisposable
+    {
+#if FEATURE_NETNATIVE //NegotiateStream
```

Comment:
As earlier comment, I would prefer WindowsSecurityToken.CoreClr.cs and WindowsSecurityToken.NetNative.cs.